source/PlayerInfo.cpp

```diff
@@ -423,23 +424,29 @@ void PlayerInfo::Load(const filesystem::path &path)
 				if(grand.Size() >= 3)
 				{
 					Date date(grand.Value(0), grand.Value(1), grand.Value(2));
-					string text;
+					// If we can find the date, use the vector we already have, other wise build one.
+					std::vector<MediaNode> &vec = logbook[date];
+
+					// Read in nodes, text or images.
```

Comment:
Is the first comment here supposed to just describe how `std::map::operator[]` works? We don't need to do that.
I don't think we need an empty line here.
I also don't know the purpose of the other comment. Reading stuff in is kind of the whole point of this method.
```suggestion
					std::vector<MediaNode> &vec = logbook[date];
```